UefiPayloadPkg/SPI/region.c

```diff
@@ -0,0 +1,498 @@
+/* SPDX-License-Identifier: GPL-2.0-only */
```

Comment:
I do not see any useage of region API, remove along with region.h @IgorBagnucki

Reply:
Deleted it